src/isomorphic/classic/types/checkReactTypeSpec.js

```diff
@@ -17,6 +17,17 @@ var ReactPropTypesSecret = require('ReactPropTypesSecret');
 var invariant = require('invariant');
 var warning = require('warning');
 
+var ReactComponentTreeDevtool;
+
+if (process.env.NODE_ENV === 'test') {
```

Comment:
I fully expect this to break internally unless we check `if (process && process.env && process.env.NODE_ENV = 'test')`.